doc/source/_toc.yml

```diff
@@ -214,6 +214,7 @@ parts:
           - file: rllib/user-guides
             sections:
               - file: rllib/rllib-models
+              - file: rllib/rllib-checkpoints-and-exports
```

Comment:
cool, thanks! @sven1977 as this is a user guide, I think you also want to add this new doc to the `panels` in `user-guides.rst`

Reply:
hmm, maybe we should take a closer look at that gallery, too. For instance, I see that "connectors" are also not in this gallery, although they show up in the TOC (and main navigation of the docs). this should align

Reply:
- was already added to user-guides.rst
- @gjoliver on adding connectors docs to list of RLlib user guides (`user-guides.rst`).